faker/providers/automotive/de_DE/__init__.py

```diff
@@ -0,0 +1,16 @@
+# coding=utf-8
+
+
+from __future__ import unicode_literals
+from .. import Provider as AutomotiveProvider
+
+
+class Provider(AutomotiveProvider):
+    license_formats = (
```

Comment:
according to https://en.wikipedia.org/wiki/Vehicle_registration_plates_of_Germany, this would generate invalid license plates. Can you write an implementation that's closer to the specification?

Reply:
@fcurella yes you're right. I changed the implementation. It generates now licence plates in the valid format with allowed prefixes.